cpp-client/cpp-tests-to-junit.sh

```diff
@@ -11,7 +11,7 @@ if [ "$#" -ne 2 ]; then
 fi
 
 if [ -z "${DH_PREFIX}" ]; then
-    echo "$0: Environment variable DHCPP_PREFIX is not set, aborting." 1>&2
+    echo "$0: Environment variable DH_PREFIX is not set, aborting." 1>&2
```

Comment:
Would `DH_CPP_PREFIX` not be more appropriate? `DH_PREFIX` is pretty generic, but since I don't really understand the full scope of this variable, feel free to say "no, I like what I did" / just resolve my comment.

Reply:
I changed it because we are installing there now more than just C++, eg, also R and potentially python/cython.